dashboard/.env.development

```diff
@@ -10,3 +10,7 @@ VITE_OIDC_ABAC_CLAIM_PATH_SEPARATOR=
 VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX=
 VITE_OIDC_ABAC_USE_ROLES=false
 VITE_OIDC_ABAC_ROLES_MAPPING=
+
+VITE_INSTITUTION_LOGO=http://localhost:8080/artefactual-logo.png
```

Comment:
:thinking: This is going to require making the CSP meta configurable too.

Reply:
Ah, right.